libs/agno/agno/playground/playground.py

```diff
@@ -32,6 +33,31 @@ def __init__(
         self.agents: Optional[List[Agent]] = agents
         self.workflows: Optional[List[Workflow]] = workflows
         self.teams: Optional[List[Team]] = teams
+        
+        if self.agents:
+            for agent in self.agents:
+                if not agent.agent_id:
+                    if agent.name:
+                        agent.agent_id = agent.name.lower().replace(" ", "-").replace("_", "-")
```

Comment:
@dirkbrnd, do you think it'd be better if we made a helper function for id generation instead of repeating it in all cases?
like this-
```py
def generate_id(name: Optional[str]) -> str:
    if name:
        return name.lower().replace(" ", "-").replace("_", "-")
    return str(uuid4())
    .....
```

not a huge thing, can skip it as well.

Reply:
Yeah good one.